### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by andreadelfante** [passsy/flutter_wrapper#13]
Hi,
it would be useful to have `flutterw.bat` for Windows platform. Essentially it can be a batch porting of the shell version.

**Comment by passsy** [comment]
I don't think this is necessary. With Windows 10 you can install Ubuntu and run the script from there.
https://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/

**Comment by sajphon** [comment]
It is necessary. I shouldn't need to install another OS just to run a command. Also, i'd like to use it from within android studio, which is not possible to do via Ubuntu under Win.

+1 for .bat

**Comment by spysoos** [comment]
+1 for .bat

**Comment by andreadelfante** [comment]
I guess we should reopen this issue.
